src/pages/RepoPage/BundlesTab/BundleOnboarding/RollupOnboarding/RollupOnboarding.spec.tsx

```diff
@@ -374,7 +376,7 @@ describe('RollupOnboarding', () => {
       render(<RollupOnboarding />, { wrapper })
 
       const gitCommit = await screen.findByText(
-        'git add -A && git commit -m "Added Codecov bundler plugin"'
+        'git add -A && git commit -m "Added Codecov bundler plugin" && git push'
```

Comment:
I feel like generally folks prefer present tense for commit messages. Definitely a nit pick though 😛 
```suggestion
        'git add -A && git commit -m "Add Codecov bundler plugin" && git push'
```